Recrutify/Recrutify.Services/Services/ProjectService.cs

```diff
@@ -53,6 +53,12 @@ public async Task<List<ProjectDTO>> GetAllAsync()
             return _mapper.Map<List<ProjectDTO>>(projects);
         }
 
+        public IQueryable<ShortProjectDTO> GetShort()
+        {
+            var projects = _projectRepository.Get();
+            return _mapper.Map<IQueryable<ShortProjectDTO>>(projects);
```

Comment:
no, see how mapping performed in another method

Reply:
done